views/slices/ImageWithCaption.ejs

```diff
@@ -0,0 +1,28 @@
+<% 
+  const imageUrl = slice.primary.image.url; 
+  const caption = ctx.prismicH.asHTML(slice.primary.caption) 
+%>
+<div class="post-part single container">
+  <% if (slice.slice_label=='image-full-width'){%>
+  <div class="blog-header single" style="background-image: url(<%=imageUrl%>)">
+    <div class="wrapper">
+      <% if(caption !== ""){%>
+      <h1>(<%- caption%>)</h1>
+      <%}%>
+    </div>
+  </div>
+  <%} else if (slice.slice_label=='emphasized'){%>
```

Comment:
1. Add spaces after the opening EJS tag and before the closing one.
2. Add spaces before and after the `==` operator.
3. Change this to a triple equals operator: `===`.

Reply:
Made all changes.